test/Microsoft.Sbom.Tool.Tests/IntegrationTests.cs

```diff
@@ -28,7 +28,7 @@ public static void Setup(TestContext context)
         testRunDirectory = context.TestRunDirectory;
     }
 
-    [ClassCleanup]
+    [ClassCleanup(ClassCleanupBehavior.EndOfClass)]
```

Comment:
FYI this can also be configured at assembly level using `[assembly: ClassCleanupExecution(ClassCleanupBehavior.EndOfClass)]` so that all class cleanup are called at the end of the class and not at the end of the assembly.